src/js/components/Sidebar.js

```diff
@@ -228,6 +163,42 @@ var Sidebar = React.createClass({
     return <ul className="sidebar-menu">{groupMenuItems}</ul>;
   },
 
+  getGroupSubmenu({path, children = []}, {currentPath, isParentActive}) {
```

Comment:
I will, though this is an old piece of code I mostly copy/pasted from the Sidebar.js